<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VizViewerWindowRotationDialog</class>
 <widget class="QDialog" name="VizViewerWindowRotationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rotate Model</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSlider" name="sliderX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QSlider" name="sliderY">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QSlider" name="sliderZ">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelX">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>25</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelY">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>25</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelZ">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>25</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save transform</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VizViewerWindowRotationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VizViewerWindowRotationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
